Source/Plugin.BLE/Android/Device.cs

```diff
@@ -412,5 +414,10 @@ protected override bool UpdateConnectionIntervalNative(ConnectionInterval interv
                 throw new Exception($"Update Connection Interval fails with error. {ex.Message}");
             }
         }
+
+        public override bool IsConnectable { get; protected set; }
+
+        public override bool CanIsConnectable { get => (Build.VERSION.SdkInt >= BuildVersionCodes.Lollipop); }
```

Comment:
Why do you use `Lollipop` here? That's Android 5 / API level 21. It actually seems like `IsConnectable` is only supported on API level 26 and above (Android 8 / Oreo). See https://developer.android.com/reference/android/bluetooth/le/ScanResult#isConnectable()

Reply:
Same reason as for the previous issue. I just looked when the interface provided ScanResult.

Reply:
I don't understand. Which interface do you mean? Can you elaborate?
As you can see from the link above, the Android API seems to support the `isConnectable` method since API level 26, no?